internal/backends/selector/Cargo.toml

```
@@ -15,16 +15,18 @@ homepage = "https://slint-ui.com"
 path = "lib.rs"
 
 [features]
-backend-gl-all = ["backend-gl-x11", "backend-gl-wayland"]
-backend-gl-wayland = ["i-slint-backend-gl/wayland"]
-backend-gl-x11 = ["i-slint-backend-gl/x11"]
+eventloop-winit = ["eventloop-winit-x11", "eventloop-winit-wayland"]
+eventloop-winit-x11 = ["i-slint-backend-winit/x11"]
+eventloop-winit-wayland = ["i-slint-backend-winit/wayland"]
 
-rtti-gl = ["i-slint-backend-gl/rtti"]
+renderer-femtovg = ["i-slint-backend-winit/renderer-femtovg"]
+
+rtti-winit = ["i-slint-backend-winit/rtti"]
 rtti-qt = ["i-slint-backend-qt/rtti"]
```

Now that we use rust 1.60, we can simplify a bit: 

```suggestion
rtti = ["i-slint-backend-winit?/rtti", "i-slint-backend-qt?/rtti"]
```